src/main/java/core/db/HOColorTable.java

```diff
@@ -0,0 +1,39 @@
+package core.db;
+
+import core.gui.theme.HOColor;
+import java.sql.Types;
+import java.util.List;
+
+public class HOColorTable extends AbstractTable {
+
+    public final static String TABLENAME = "HOColor";
+
+    /**
+     * constructor
+     *
+     * @param connectionManager Connection manager
+     */
+    public HOColorTable(ConnectionManager connectionManager) {
+        super(TABLENAME, connectionManager);
+        idColumns = 2;
+    }
+
+    @Override
+    protected void initColumns() {
+        columns = new ColumnDescriptor[]{
+                ColumnDescriptor.Builder.newInstance().setColumnName("NAME").setGetter((o) -> ((HOColor) o).getName()).setSetter((o, v) -> ((HOColor) o).setName((String) v)).setType(Types.VARCHAR).isNullable(false).setLength(255).build(),
+                ColumnDescriptor.Builder.newInstance().setColumnName("THEME").setGetter((o) -> ((HOColor) o).getTheme()).setSetter((o, v) -> ((HOColor) o).setTheme((String) v)).setType(Types.VARCHAR).isNullable(false).setLength(255).build(),
+                ColumnDescriptor.Builder.newInstance().setColumnName("COLOR_REFERENCE").setGetter((o) -> ((HOColor) o).getColorReference()).setSetter((o, v) -> ((HOColor) o).setColorReference((String) v)).setType(Types.VARCHAR).isNullable(true).setLength(255).build(),
+                ColumnDescriptor.Builder.newInstance().setColumnName("VALUE").setGetter((o) -> ((HOColor) o).getValue()).setSetter((o, v) -> ((HOColor) o).setValue((Integer) v)).setType(Types.INTEGER).isNullable(true).build(),
+        };
+    }
+
+    @Override
+    protected String createSelectStatement() {
+        return createSelectStatement("WHERE THEME=?");
```

Comment:
Do we need an index on `THEME`?

Reply:
I haven't thought about it yet. 
However, I assume that the normal case will be that only a single value will be stored in the column.
Isn't an index more of a hindrance?

Reply:
A hindrance, in what way?  

If we don't expect that table to contain many records, then no need for an index.